examples/CMakeLists.txt

```
@@ -20,4 +20,8 @@ endif ()
 if (OpenCV_FOUND)
     add_subdirectory(green_screen)
     add_subdirectory(opencv_compatibility)
+
+    if ("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+        file(COPY ${OpenCV_DIR}/../bin DESTINATION ${CMAKE_BINARY_DIR} FILES_MATCHING PATTERN "*.dll")
```

I think we actually want to use `${CMAKE_RUNTIME_OUTPUT_DIRECTORY}`
```suggestion
        file(COPY ${OpenCV_DIR}/../bin DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY} FILES_MATCHING PATTERN "*.dll")
```

`${CMAKE_BINARY_DIR}` is just the /build/ folder, `${CMAKE_RUNTIME_OUTPUT_DIRECTORY}` is the /build/bin folder